src/assets/locales/de/translations.json

```diff
@@ -125,17 +125,18 @@
   "services.platform.list.designsystems": "Design Systeme",
   "services.platform.list.infrastructure": "Infrastruktur & Automation",
   "services.consulting.title": "Beratung",
-  "services.consulting.teaser": "Die Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz.",
-  "services.consulting.text": "Die Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz. Wir\n          teilen diesen sehr gerne mit Ihnen und beraten Sie in technischen und\n          konzeptionellen Angelegenheiten.",
+  "services.consulting.teaser": "Die langjährige Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz.",
+  "services.consulting.text": "Die langjährige Erfahrung unserer Mitarbeiter ist unser wertvollster Schatz. Wir\n          teilen unser Wissen sehr gerne mit Ihnen und beraten Sie in technischen und\n          konzeptionellen Angelegenheiten.",
```

Comment:
The `\n` line breaks don't seem to work.
![image](https://user-images.githubusercontent.com/7970458/146551135-3b1a1311-149e-4144-bb71-38bf20a80b43.png)
Did they work in the past? Do we want them to work? If not we might as well remove them.